CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.1.0
+
+* Update GitPython to 3.1.40.
+* Change Python version for CI Test from 3.6 to 3.8.
```

Comment:
Please also note explicitly that the python 3.6 support was dropped (it's not just about the CI)